### PR TITLE
New Cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR has mostly cruise-specific cal csvs for the Irminger 9 cruise, plus an OPTAA that just returned from service.

### CODE REVIEW

File: calibration/ADCPSL/CGINS-ADCPSL-20504__20220620.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20504,CC_scale_factor1,0.45,constant; GI03FLMB-00009
+20504,CC_scale_factor2,0.45,constant
+20504,CC_scale_factor3,0.45,constant
+20504,CC_scale_factor4,0.45,constant
+20504,CC_bin_size,1000.0,in cm
+20504,CC_dist_first_bin,1647.0,in cm
+20504,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (snwhite) [inline calibration/ADCPSL/CGINS-ADCPSL-20504__20220620.csv:8]: Looks good.

File: calibration/ADCPSL/CGINS-ADCPSL-23498__20220620.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+23498,CC_scale_factor1,0.45,constant; GI03FLMA-00009
+23498,CC_scale_factor2,0.45,constant
+23498,CC_scale_factor3,0.45,constant
+23498,CC_scale_factor4,0.45,constant
+23498,CC_bin_size,1000.0,in cm
+23498,CC_dist_first_bin,1647.0,in cm
+23498,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (snwhite) [inline calibration/ADCPSL/CGINS-ADCPSL-23498__20220620.csv:8]: Looks good.

File: calibration/ADCPSN/CGINS-ADCPSN-22865__20220620.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+22865,CC_scale_factor1,0.45,constant; GI01SUMO-00009
+22865,CC_scale_factor2,0.45,constant
+22865,CC_scale_factor3,0.45,constant
+22865,CC_scale_factor4,0.45,constant
+22865,CC_bin_size,1000.0,in cm
+22865,CC_dist_first_bin,1647.0,in cm
+22865,CC_orientation,1.0,1 is upward looking; 0 is downward looking
```
Comment (snwhite) [inline calibration/ADCPSN/CGINS-ADCPSN-22865__20220620.csv:8]: Looks good.

File: calibration/METBKA/CGINS-METLGR-00016__20220620.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR016,CC_depth_of_conductivity_and_temperature_measurements_m,1.23,GI01SUMO-00009 METBK1
+LGR016,CC_height_of_air_humidity_measurement_m,5.255,
+LGR016,CC_height_of_air_temperature_measurement_m,5.255,
+LGR016,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
+LGR016,CC_jcool,1,Constant
+LGR016,CC_jwarm,1,Constant
+LGR016,CC_zinvpbl,600,Constant
+LGR016,CC_use_velpt,1,
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00016__20220620.csv:9]: Looks good.

File: calibration/METBKA/CGINS-METLGR-00031__20220620.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+LGR031,CC_depth_of_conductivity_and_temperature_measurements_m,1.37,GI01SUMO-00009 METBK2
+LGR031,CC_height_of_air_humidity_measurement_m,5.255,
+LGR031,CC_height_of_air_temperature_measurement_m,5.255,
+LGR031,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
+LGR031,CC_jcool,1,Constant
+LGR031,CC_jwarm,1,Constant
+LGR031,CC_zinvpbl,600,Constant
+LGR031,CC_use_velpt,1,
```
Comment (snwhite) [inline calibration/METBKA/CGINS-METLGR-00031__20220620.csv:9]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01088__20220531.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
...73.7, 274.52, 275.34, 276.16, 276.99, 277.81, 278.63, 279.45, 280.27, 281.1, 281.92, 282.74, 283.56, 284.38, 285.21, 286.03, 286.85, 287.67, 288.49, 289.31, 290.14, 290.96, 291.78, 292.6, 293.42, 294.24, 295.07, 295.89, 296.71, 297.53, 298.35, 299.17, 299.99, 300.81, 301.63, 302.46, 303.28, 304.1, 304.92, 305.74, 306.56, 307.38, 308.2, 309.02, 309.84, 310.66, 311.48, 312.3, 313.12, 313.94, 314.76, 315.58, 316.4, 317.22, 318.04, 318.86, 319.68, 320.5, 321.32, 322.13, 322.95, 323.77, 324.59, 325.41, 326.23, 327.04, 327.86, 328.68, 329.5, 330.32, 331.13, 331.95, 332.77, 333.58, 334.4, 335.22, 336.04, 336.85, 337.67, 338.48, 339.3, 340.12, 340.93, 341.75, 342.56, 343.38, 344.19, 345.01, 345.82, 346.64, 347.45, 348.27, 349.08, 349.89, 350.71, 351.52, 352.34, 353.15, 353.96, 354.77, 355.59, 356.4, 357.21, 358.02, 358.84, 359.65, 360.46, 361.27, 362.08, 362.89, 363.7, 364.51, 365.32, 366.13, 366.94, 367.75, 368.56, 369.37, 370.18, 370.99, 371.8, 372.61, 373.41, 374.22, 375.03, 375.84, 376.64, 377.45, 378.26, 379.06, 379.87, 380.67, 381.48, 382.29, 383.09, 383.89, 384.7, 385.5, 386.31, 387.11, 387.91, 388.72, 389.52, 390.32, 391.13, 391.93, 392.73, 393.53, 394.33, 395.13, 395.93, 396.73]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01088__20220531.csv:8]: Spot checked beginning and end.  Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01089__20220531.csv [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
...273.48, 274.28, 275.08, 275.88, 276.68, 277.48, 278.28, 279.08, 279.88, 280.69, 281.49, 282.29, 283.09, 283.89, 284.69, 285.49, 286.29, 287.09, 287.89, 288.69, 289.49, 290.29, 291.09, 291.89, 292.69, 293.49, 294.29, 295.09, 295.89, 296.69, 297.49, 298.29, 299.09, 299.89, 300.69, 301.49, 302.29, 303.09, 303.89, 304.68, 305.48, 306.28, 307.08, 307.88, 308.68, 309.48, 310.28, 311.08, 311.88, 312.68, 313.47, 314.27, 315.07, 315.87, 316.67, 317.47, 318.27, 319.06, 319.86, 320.66, 321.46, 322.26, 323.05, 323.85, 324.65, 325.45, 326.24, 327.04, 327.84, 328.63, 329.43, 330.23, 331.03, 331.82, 332.62, 333.41, 334.21, 335.01, 335.8, 336.6, 337.39, 338.19, 338.99, 339.78, 340.58, 341.37, 342.17, 342.96, 343.76, 344.55, 345.34, 346.14, 346.93, 347.73, 348.52, 349.31, 350.11, 350.9, 351.69, 352.49, 353.28, 354.07, 354.87, 355.66, 356.45, 357.24, 358.03, 358.83, 359.62, 360.41, 361.2, 361.99, 362.78, 363.57, 364.36, 365.15, 365.94, 366.73, 367.52, 368.31, 369.1, 369.89, 370.68, 371.47, 372.26, 373.04, 373.83, 374.62, 375.41, 376.2, 376.98, 377.77, 378.56, 379.34, 380.13, 380.91, 381.7, 382.49, 383.27, 384.06, 384.84, 385.63, 386.41, 387.2, 387.98, 388.76, 389.55, 390.33, 391.11, 391.9, 392.68]",
```
Comment (snwhite) [inline calibration/NUTNRB/CGINS-NUTNRB-01089__20220531.csv:8]: Spot checked beginning and end.  Looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00760__20220531.csv [hunk 8]
```diff
@@ -0,0 +1,8 @@
...
...273.71, 274.52, 275.32, 276.13, 276.94, 277.75, 278.56, 279.37, 280.18, 280.99, 281.8, 282.61, 283.42, 284.23, 285.04, 285.85, 286.65, 287.46, 288.27, 289.08, 289.89, 290.7, 291.51, 292.32, 293.13, 293.94, 294.75, 295.56, 296.37, 297.18, 297.99, 298.8, 299.61, 300.42, 301.23, 302.04, 302.84, 303.65, 304.46, 305.27, 306.08, 306.89, 307.7, 308.51, 309.32, 310.13, 310.94, 311.75, 312.56, 313.37, 314.17, 314.98, 315.79, 316.6, 317.41, 318.22, 319.03, 319.84, 320.64, 321.45, 322.26, 323.07, 323.88, 324.69, 325.49, 326.3, 327.11, 327.92, 328.73, 329.53, 330.34, 331.15, 331.96, 332.76, 333.57, 334.38, 335.19, 335.99, 336.8, 337.61, 338.41, 339.22, 340.03, 340.83, 341.64, 342.45, 343.25, 344.06, 344.86, 345.67, 346.48, 347.28, 348.09, 348.89, 349.7, 350.5, 351.31, 352.11, 352.92, 353.72, 354.53, 355.33, 356.13, 356.94, 357.74, 358.55, 359.35, 360.15, 360.96, 361.76, 362.56, 363.36, 364.17, 364.97, 365.77, 366.57, 367.38, 368.18, 368.98, 369.78, 370.58, 371.38, 372.18, 372.98, 373.79, 374.59, 375.39, 376.19, 376.99, 377.79, 378.58, 379.38, 380.18, 380.98, 381.78, 382.58, 383.38, 384.18, 384.97, 385.77, 386.57, 387.37, 388.16, 388.96, 389.76, 390.55, 391.35, 392.14, 392.94, 393.73, 394.53]",
```
Comment (snwhite) [inline calibration/NUTNRM/CGINS-NUTNRM-00760__20220531.csv:8]: Spot checked beginning and end.  Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00152__20220520.csv [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+ACS-152,CC_tbins,"[0.958655, 1.264689, 2.391053, 3.484643, 4.497895, 5.598500, 6.490000, 7.530000, 8.526000, 9.514583, 10.490417, 11.482800, 12.490000, 13.493200, 14.484167, 15.493462, 16.517200, 17.474400, 18.492800, 19.505417, 20.508462, 21.497083, 22.488462, 23.508077, 24.518800, 25.506800, 26.519000, 27.490769, 28.497037, 29.506667, 30.498400, 31.473846, 32.488519, 33.516786, 34.518966]",
+ACS-152,CC_taarray,SheetRef:CC_taarray,
+ACS-152,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00152__20220520.csv:9]: Spot checked beginning and end.  Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00152__20220520__CC_tcarray.ext [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+0.004615,0.003540,0.000682,-0.000028,0.000095,0.000057,0.000018,0.000174,0.000414,0.000561,0.000409,0.000580,0.000747,0.000626,0.000627,0.000641,0.000666,0.000655,0.000814,0.000547,0.000536,0.000469,0.000415,0.000239,0.000220,0.000000,-0.000080,-0.000266,-0.000583,-0.000690,-0.001054,-0.001385,-0.001532,-0.001711,-0.001927
+0.004503,0.003582,0.000731,0.000292,0.000178,-0.000014,0.000169,0.000311,0.000316,0.000510,0.000358,0.000519,0.000485,0.000606,0.000796,0.000623,0.000682,0.000938,0.000894,0.000587,0.000604,0.000428,0.000430,0.000411,0.000226,0.000000,-0.000020,-0.000292,-0.000506,-0.000563,-0.000936,-0.000954,-0.001335,-0.001383,-0.001840
+0.002181,0.001587,-0.000571,-0.001132,-0.000828,-0.000818,-0.000546,-0.000242,-0.000099,0.000198,0.000072,0.000399,0.000382,0.000519,0.000535,0.000599,0.000461,0.000504,0.000688,0.000441,0.000489,0.000367,0.000386,0.000124,0.000091,0.000000,-0.000173,-0.000368,-0.000689,-0.000913,-0.001295,-0.001736,-0.001845,-0.002001,-0.002463
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00152__20220520__CC_tcarray.ext:83]: Spot checked beginning and end.  Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00152__20220520__CC_taarray.ext [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+0.004904,0.004462,0.003047,0.002386,0.002228,0.001757,0.000959,0.001170,0.000943,0.000755,0.000683,0.000527,0.000636,0.000409,0.000477,0.000434,0.000474,0.000306,-0.000045,0.000044,0.000379,0.000288,0.000183,-0.000074,-0.000061,0.000000,-0.000278,-0.000171,-0.000572,-0.000532,-0.000792,-0.001141,-0.001255,-0.001551,-0.001678
+0.005010,0.004603,0.003155,0.002351,0.002356,0.001929,0.001118,0.001300,0.001117,0.000871,0.000932,0.000766,0.000747,0.000503,0.000643,0.000609,0.000546,0.000429,0.000122,0.000182,0.000431,0.000486,0.000381,0.000011,0.000038,0.000000,-0.000085,-0.000106,-0.000416,-0.000431,-0.000770,-0.000997,-0.000927,-0.001464,-0.001494
+0.004710,0.004274,0.002868,0.002076,0.002066,0.001680,0.000761,0.001074,0.000946,0.000748,0.000679,0.000582,0.000634,0.000389,0.000531,0.000515,0.000627,0.000349,0.000107,0.000199,0.000422,0.000460,0.000399,0.000084,0.000012,0.000000,-0.000155,-0.000116,-0.000438,-0.000474,-0.000816,-0.001049,-0.001106,-0.001711,-0.001646
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00152__20220520__CC_taarray.ext:83]: Spot checked beginning and end.  Looks good.

**Review by snwhite — APPROVED** [review]

File: calibration/ADCPSL/CGINS-ADCPSL-20504__20220620.csv (hunk 1 above)
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-20504__20220620.csv:8]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-23498__20220620.csv (hunk 2 above)
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-23498__20220620.csv:8]: looks good

File: calibration/ADCPSN/CGINS-ADCPSN-22865__20220620.csv (hunk 3 above)
Comment (cdobs) [inline calibration/ADCPSN/CGINS-ADCPSN-22865__20220620.csv:8]: looks good

File: calibration/METBKA/CGINS-METLGR-00016__20220620.csv (hunk 4 above)
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00016__20220620.csv:9]: looks good

File: calibration/METBKA/CGINS-METLGR-00031__20220620.csv (hunk 5 above)
Comment (cdobs) [inline calibration/METBKA/CGINS-METLGR-00031__20220620.csv:9]: looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01088__20220531.csv (hunk 6 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01088__20220531.csv:8]: Spot check - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01089__20220531.csv (hunk 7 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01089__20220531.csv:8]: spot check - looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00760__20220531.csv (hunk 8 above)
Comment (cdobs) [inline calibration/NUTNRM/CGINS-NUTNRM-00760__20220531.csv:8]: spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00152__20220520.csv (hunk 9 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00152__20220520.csv:9]: spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00152__20220520__CC_taarray.ext (hunk 11 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00152__20220520__CC_taarray.ext:83]: spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00152__20220520__CC_tcarray.ext (hunk 10 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00152__20220520__CC_tcarray.ext:83]: spot check - looks good

**Review by cdobs — APPROVED** [review]
looks good!

**Pull request merged**